### PR TITLE
Move login.css and setup.css to system

### PR DESCRIPTION
**Pull request by ylavoie**
Make sure that all CSS are included by webpack

### CODE REVIEW

File: UI/webpack.config.js [hunk 1]
```diff
@@ -65,6 +65,20 @@ if (TARGET !== "readme") {
             .sort();
     }
 
+    /* eslint-disable-next-line no-inner-declarations */
+    function globCssEntries(globPath) {
+        var files = glob.sync(globPath);
+        var entries = {};
```
Comment (ehuelsmann) [inline UI/webpack.config.js:71]: Using `let` should eliminate inner declaration warnings or errors.

**Review by ehuelsmann — APPROVED** [review]
Looks good. Can we backport to 1.10 and 1.9 ?

File: UI/webpack.config.js [hunk 2]
```diff
@@ -67,11 +67,11 @@ if (TARGET !== "readme") {
 
     /* eslint-disable-next-line no-inner-declarations */
     function globCssEntries(globPath) {
-        var files = glob.sync(globPath);
-        var entries = {};
```
Comment (ehuelsmann) [inline UI/webpack.config.js]: With this change, we can remove the "no-inner-declaratios" eslint directive, no?

Comment (ylavoie) [reply UI/webpack.config.js]: No, the function is not defined at root but in the  application section, because README has no use for that.

**Pull request merged**